students/20th/garamsong/login/login.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="utf-8">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href = "./common.css" rel = "stylesheet" type = "text/css">
+        <link href = "./login.css" rel = "stylesheet" type = "text/css">
+        <title>westagram</title>
+    </head>
+    <body>
+        <section class = "loginContainer">
+            <div class = "loginLogoContainer">
```

Comment:
```suggestion
        <section class="loginContainer">
            <div class="loginLogoContainer">
```
속성과 등호를 붙여 쓰는건 어떨까요?

Reply:
확인 후 수정완료 했습니다!